### ISSUE TITLE
Does it work with gray images?

### ISSUE THREAD

**Issue by AldrichCabrera** [alexgkendall/caffe-posenet#35]
I have trained the network with 800 epoch using gray images but the prediction is incorrect and the average error is 2,086 m in an interior area of ​​3.5 meters. My question is whether it works with gray images or should I make any changes?

Regards,

**Comment by DRAhmadFaraz** [comment]
@AldrichCabrera No, It would not work on gray scale images, You should have to put 3 channel RGB image in network.
